sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveExternalCatalogVersionsSuite.scala

```diff
@@ -85,6 +90,43 @@ class HiveExternalCatalogVersionsSuite extends SparkSubmitTestUtils {
     new File(tmpDataDir, name).getCanonicalPath
   }
 
+  private def getFileFromUrl(urlString: String, targetDir: String, filename: String): Boolean = {
+    val conf = new SparkConf
+    val securityManager = new SecurityManager(conf)
+    val hadoopConf = new Configuration
+
+    val outDir = new File(targetDir)
+    if (!outDir.exists()) {
+      outDir.mkdirs()
+    }
+
+    try {
+      val result = Utils.doFetchFile(urlString, outDir, filename, conf, securityManager, hadoopConf)
```

Comment:
`Utils.doFetchFile` is a little overkill for this case, maybe we can just implement a simple downloading function with java.

Reply:
Meh, if we can reuse code instead of rewriting it, seems OK to me